<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Location</class>
 <widget class="QDialog" name="Location">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>incidents at this location</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>511</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(49, 145, 158);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="lbSearch">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string>enter coutry:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leSearch">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbTitle">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>231</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Ubuntu&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Location information</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>0</y>
      <width>145</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0,0">
     <property name="spacing">
      <number>1</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <item>
      <widget class="QPushButton" name="btnDel">
       <property name="maximumSize">
        <size>
         <width>35</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(238, 238, 236);</string>
       </property>
       <property name="text">
        <string>del</string>
       </property>
       <property name="iconSize">
        <size>
         <width>7</width>
         <height>8</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAdd">
       <property name="maximumSize">
        <size>
         <width>35</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(238, 238, 236);</string>
       </property>
       <property name="text">
        <string>add</string>
       </property>
       <property name="iconSize">
        <size>
         <width>7</width>
         <height>8</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnEdit">
       <property name="maximumSize">
        <size>
         <width>35</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(238, 238, 236);</string>
       </property>
       <property name="text">
        <string>edit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnBack">
       <property name="maximumSize">
        <size>
         <width>35</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(238, 238, 236);
</string>
       </property>
       <property name="text">
        <string>back</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>50</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>find</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>251</width>
     <height>58</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>country</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="leCountry"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>water body</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="leWaterBody"/>
    </item>
   </layout>
  </widget>
  <widget class="QTableWidget" name="twIncidents">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>170</y>
     <width>281</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnNext">
   <property name="geometry">
    <rect>
     <x>438</x>
     <y>130</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(49, 145, 158);
</string>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPrev">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>130</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(49, 145, 158);
</string>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
